Update connecting-your-database.md

mirnawong1 opened the pull request:
adding comment to instruct users to add commented lines when adding private key field:


:::**Note:** At this time ONLY Encrypted Private Keys are supported by dbt Cloud. In order to successfully fill in the Private Key field, you  **must** include the commented lines below when you add the passphrase. The passphrase you add, will be used to decrypt the and leaving the `PRIVATE KEY PASSPHRASE` field empty will will return an error.
:::
**Example:**
```sql
-----BEGIN ENCRYPTED PRIVATE KEY-----
< encrypted private key contents here >
-----END ENCRYPTED PRIVATE KEY-----
```
## Description & motivation
<!---
Describe your changes, and why you're making them. Is this linked to an open
issue, a pull request on dbt core, etc?
-->

## To-do before merge
<!---
(Optional -- remove this section if not needed)
Include any notes about things that need to happen before this PR is merged, e.g.:
- [ ] Change the base branch
- [ ] Ensure PR #56 is merged
-->

## Pre-release docs
Is this change related to an unreleased version of dbt?
- [ ] Yes: please
    - update the base branch to `next`
    - add Changelog components: `<Changelog>[New/Changed] in v0.x.0</Changelog>`
    - add links to the "New and changed documentation" section of the latest [Migration Guide](../website/docs/docs/guides/migration-guide)
- [x ] No: please ensure the base branch is `current`
- [ ] Unsure: we'll let you know!

## Checklist
If you added new pages (delete if not applicable):
- [ ] The page has been added to `website/sidebars.js`
- [ ] The new page has a unique filename

If you removed existing pages (delete if not applicable):
- [ ] The page has been removed from `website/sidebars.js`
- [ ] An entry has been added to `_redirects`

mirnawong1 committed:
Update connecting-your-database.md

adding comment to instruct users to add commented lines when adding private key field:


:::**Note:** At this time ONLY Encrypted Private Keys are supported by dbt Cloud. In order to successfully fill in the Private Key field, you  **must** include the commented lines below when you add the passphrase. The passphrase you add, will be used to decrypt the and leaving the `PRIVATE KEY PASSPHRASE` field empty will will return an error.
:::
**Example:**
```sql
-----BEGIN ENCRYPTED PRIVATE KEY-----
< encrypted private key contents here >
-----END ENCRYPTED PRIVATE KEY-----

krevitt commented on website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-your-database.md:
```diff
@@ -83,7 +83,15 @@ Finally, set the "Private Key" and "Private Key Passphrase" fields in the "Edit
 using a key pair.
 
-**Note:** At this time ONLY Encrypted Private Keys are supported by dbt Cloud -- you **must** add the passphrase that will be used to decrypt the and will receive an error if the `PRIVATE KEY PASSPHRASE` field is empty.
+:::**Note:** At this time ONLY Encrypted Private Keys are supported by dbt Cloud. In order to successfully fill in the Private Key field, you  **must** include the commented lines below when you add the passphrase. The passphrase you add, will be used to decrypt the and leaving the `PRIVATE KEY PASSPHRASE` field empty will will return an error.
```
In the [deploy preview](https://deploy-preview-1183--docs-getdbt-com.netlify.app/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-your-database), looks like the formatting of this section is a bit off.  

Think you can remove the `:::` from before and after the note.

krevitt replied:
Fixed this

krevitt commented on website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-your-database.md:
````diff
@@ -83,7 +83,15 @@ Finally, set the "Private Key" and "Private Key Passphrase" fields in the "Edit
+
+
+**Example:**
+```sql
````
Will want to close this ```sql out with an ending ```!  Currently the rest of the page is being pulled into this SQL block.

krevitt replied:
Fixed this

krevitt committed:
Updated formatting

krevitt committed:
Merge branch 'current' into mirnawong1-patch-18

krevitt commented on website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-your-database.md:
```diff
@@ -83,7 +83,15 @@ Finally, set the "Private Key" and "Private Key Passphrase" fields in the "Edit
 using a key pair.
 
-**Note:** At this time ONLY Encrypted Private Keys are supported by dbt Cloud -- you **must** add the passphrase that will be used to decrypt the and will receive an error if the `PRIVATE KEY PASSPHRASE` field is empty.
+**Note:** At this time ONLY Encrypted Private Keys are supported by dbt Cloud. In order to successfully fill in the Private Key field, you  **must** include the commented lines below when you add the passphrase. The passphrase you add, will be used to decrypt the and leaving the `PRIVATE KEY PASSPHRASE` field empty will will return an error.
```
The 2nd sentence here might be incomplete - "the passphrase you add, will be used to decrypt the {{ missing word, key? }} and".

Also looks like maybe that should be a line break from the "Note" about the commented lines.

Otherwise looks good to go!

mirnawong1 replied:
hi @krevitt that's a great callout and thank you so much for those suggestions! I've made those [changes](https://github.com/dbt-labs/docs.getdbt.com/pull/1183/commits/e04e82c7d1a736b5e98aad32708962a504f1ad7c) and also removed the duplicate 'will' at the end as well. hope this is good to go now and appreciate the help!

krevitt reviewed (changes requested):
hey @mirnawong1, looks good!  left one comment of word choice to fix up, but otherwise LGTM

mirnawong1 committed:
Update connecting-your-database.md

hi @krevitt that's a great callout and thank you so much for those suggestions! i've made those changes and also removed the duplicate 'will' at the end as well. hope this is good to go now and appreciate the help!

mirnawong1 committed:
Merge branch 'current' into mirnawong1-patch-18

mirnawong1 committed:
Update connecting-your-database.md

krevitt committed:
Our example -> the examples

Little syntax thing - think the goal is to use 2nd person (non possessive) in docs writing

krevitt reviewed (approved):
LGTM!  Feel free to merge in

mirnawong1 committed:
Merge branch 'current' into mirnawong1-patch-18

mirnawong1 commented:
thank you @krevitt !
